README.md

```diff
@@ -49,10 +49,10 @@ This is an example configuration that should cover most relevant aspects of the
 ```yaml
 global:
   # The smarthost and SMTP sender used for mail notifications.
-  # If the host requires authentication, set SMTP_AUTH_USERNAME and SMTP_AUTH_PASSWORD
-  # environment variables.
   smtp_smarthost: 'localhost:25'
   smtp_from: 'alertmanager@example.org'
+  smtp_auth_username: 'alertmanager'
```

Comment:
With this in the config now, we can remove this from here and just have it in the docs and simple.yml.